### PR TITLE
Moving scalafmt to a non-RC version.

### PR DESCRIPTION
**Pull request by kshakir**
### Description

The IntelliJ plugin moved from a release candidate of scalafmt 1.0.0 to 1.1.0. Updated the scalafmt used by clio to a stable release, but cannot move to neo-sbt-scalafmt latest and scalafmt latest just yet because of various issues.

----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: Ensure that you have added or updated tests
- [x] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=114531509).
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
* Review cycle:
  * Team may comment on PR at will
  * **Reviewer assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to reviewer** for further feedback
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Move the JIRA issue to QA once this checklist is completed

### CODE REVIEW

**Comment by broadinst** [comment]
Can one of the admins verify this patch?

**Comment by pshapiro4broad** [comment]
add to whitelist

**Comment by pshapiro4broad** [comment]
retest this please

File: project/project/plugins.sbt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-val ScalafmtVersion = "1.7"
+val ScalafmtVersion = "1.7" // https://github.com/lucidsoftware/neo-sbt-scalafmt/issues/31
```
Comment (pshapiro4broad) [inline project/project/plugins.sbt:1]: Do we have a way to share these version numbers between sbt modules?

**Review by pshapiro4broad — APPROVED** [review]
Am hoping that this isn't a sign that scalafmt isn't stable enough to use..

👍

File: project/project/plugins.sbt (hunk 1 above)
Comment (kshakir) [reply project/project/plugins.sbt:1]: In theory, one could write a bit of sbt DSL code that during the meta-build of `/project/project`, it **writes** a file with these variables, and then the main build under `/project` **reads** the generated file.

I looked at a couple of stack-overflow [options](https://stackoverflow.com/questions/23944108/how-to-share-version-values-between-project-plugins-sbt-and-project-build-scala) and hated all of the suggestions. Possibly could be done in a future ticket though?

**Pull request merged**